### ISSUE TITLE
Very Low mAP

### ISSUE THREAD

**Issue by mahiratmis** [visinf/1-stage-wseg#19]
Hi, I downloaded your model (**model_enc_e020Xs0.928.pth**) and started training with it. I ran _bash ./launch/run_voc_resnet38.sh_  Training started by printing the following statistics:
**....
Group 0: #35, LR=1.000e-03
Group 1: #0, LR=2.000e-03
Group 2: #26, LR=1.000e-02
Group 3: #13, LR=2.000e-02
Using SGD >>> learning rate = 1.000e-03, momentum = 9.000e-01, weight decay = 5.000e-04
Epoch >>>  0
AP_aeroplane: 0.063
AP_bicycle: 0.040
AP_bird: 0.054
AP_boat: 0.048
AP_bottle: 0.063
AP_bus: 0.036
AP_car: 0.105
AP_cat: 0.054
AP_chair: 0.082
AP_cow: 0.086
AP_diningtable: 0.097
AP_dog: 0.101
AP_horse: 0.094
AP_motorbike: 0.083
AP_person: 0.346
AP_potted-plant: 0.068
AP_sheep: 0.049
AP_sofa: 0.063
AP_train: 0.037
AP_tv/monitor: 0.047
mAP: 0.081
Validation / Train:  2.1m
AP_aeroplane: 0.052
AP_bicycle: 0.049
AP_bird: 0.052
AP_boat: 0.047
AP_bottle: 0.064
AP_bus: 0.035
AP_car: 0.112
AP_cat: 0.052
AP_chair: 0.070
AP_cow: 0.090
AP_diningtable: 0.090
AP_dog: 0.117
AP_horse: 0.152
AP_motorbike: 0.082
AP_person: 0.345
AP_potted-plant: 0.059
AP_sheep: 0.061
AP_sofa: 0.064
AP_train: 0.039
AP_tv/monitor: 0.041
mAP: 0.084
Validation /   Val:  2.1m
New Epoch:  Mon Jan 11 21:40:24 2021**

mAP seems to be 0.084. Will it get better with more epochs. Am I missing something here?
The config is as follows:
**Config: 
 {'NUM_GPUS': 1, 'TRAIN': {'BATCH_SIZE': 8, 'NUM_EPOCHS': 24, 'NUM_WORKERS': 8, 'MASK_LOSS': 0.0, 'PRETRAIN': 5}, 'TEST': {'METHOD': 'multiscale', 'DATA_ROOT': '/fastdata/naraslanov', 'SCALES': [1, 0.5, 1.5, 2.0], 'FLIP': True, 'PAD_SIZE': [1024, 1024], 'CROP_SIZE': [448, 448], 'CROP_GRID_SIZE': [2, 2], 'BATCH_SIZE': 8, 'BG_POW': 3, 'NUM_CLASSES': 21, 'USE_GT_LABELS': False, 'FP_CUT_SCORE': 0.1}, 'DATASET': {'CROP_SIZE': 321, 'SCALE_FROM': 0.9, 'SCALE_TO': 1.0, 'PATH': 'data/images'}, 'NET': {'MODEL': 'ae', 'BACKBONE': 'resnet38', 'PRE_WEIGHTS_PATH': './models/weights/model_enc_e020Xs0.928.pth', 'OPT': 'SGD', 'LR': 0.001, 'BETA1': 0.5, 'MOMENTUM': 0.9, 'WEIGHT_DECAY': 0.0005, 'LOSS': 'SoftMargin', 'MASK_LOSS_BCE': 1.0, 'BG_SCORE': 0.1, 'FOCAL_P': 3, 'FOCAL_LAMBDA': 0.01, 'PAMR_KERNEL': [1, 2, 4, 8, 12, 24], 'PAMR_ITER': 10, 'SG_PSI': 0.3}, 'MASKS': {'CRF': {'ALPHA_LOW': 4, 'ALPHA_HIGH': 32}}, 'CUDA': False, 'DEBUG': False, 'PYTORCH_VERSION_LESS_THAN_040': False}
Backbone: ResNet38
Model: AE
Loading weights from:  ./models/weights/model_enc_e020Xs0.928.pth**

**Comment by mahiratmis** [comment]
It did reach pretty high scores thank you.
